tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/util/ProjectImportUtil.java

```diff
@@ -500,6 +500,63 @@ else if( tempRecord.isTheme() )
         return isValid;
     }
 
+    public static IStatus validateSDKPath(final String currentPath)
+    {
+        IStatus retVal = Status.OK_STATUS;
+
+        if( !org.eclipse.core.runtime.Path.EMPTY.isValidPath( currentPath ) )
+        {
+            retVal = ProjectCore.createErrorStatus( "\"" + currentPath + "\" is not a valid path." );
+        }
+        else
+        {
+            IPath osPath = org.eclipse.core.runtime.Path.fromOSString( currentPath );
+
+            if( !osPath.toFile().isAbsolute() )
+            {
+                retVal = ProjectCore.createErrorStatus( "\"" + currentPath + "\" is not an absolute path." );
```

Comment:
sdk path doesn't have to be absolute, it can be relative.  Often the paths are indeed relative.  Look at the liferay-plugins-sdk default value.  The default value of app.server.parent.dir is "${sdk.dir}/.../bundles"  So if a user has added their bundle to this location then we need to support this.  Doesn't the SDK's own validate method already do this?  Also we can just re-use the SDK's class own validate method instead of adding more logic here right?